docs/config/_default/config.toml

```diff
@@ -1,6 +1,6 @@
 # Production BaseURL
-#baseURL = "https://pgjdbc.github.io/pgjdbcdocs/"
-baseURL = "https://jdbc.postgresql.org/"
+baseURL = "https://pgjdbc.github.io/pgjdbc/"
```

Comment:
Do you intend moving all the documentation to the new URL?

Reply:
Looking at the docs I don't see a way to publish to the root of https://pgjdbc.github.io/ it looks like it requires https://pgjdbc.github.io/pgjdbc as there could be many projects under the pgjdbc project

Reply:
Have you considered testing in your fork like davecramer.github.io/pgjdbc? I guess you can test in your fork. There's no much benefit from merging the change that will not be used later.